src/JustSaying/JustSaying.csproj

```diff
@@ -6,8 +6,8 @@
     <ProjectReference Include="..\JustSaying.Models\JustSaying.Models.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.3.100.1" />
-    <PackageReference Include="AWSSDK.SQS" Version="3.3.100.1" />
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.5.0" />
+    <PackageReference Include="AWSSDK.SQS" Version="3.5.0" />
```

Comment:
I presume this is deliberately different from `AwsSdkSqsVersion` as this is intended to be the minimum supported version?

Reply:
Yeah I "pinned" this for the libraries, but the other is for the tests to use "latest"